.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .idea
-lib/**/*
\ No newline at end of file
+lib/**/*
+.secrets
```

Comment:
The addition of '.secrets' to the .gitignore is a good practice to ensure sensitive information is not tracked by Git. Ensure that any secrets or sensitive files are properly managed and not accidentally committed.